lessons/en/basics/date_time.md

```diff
@@ -100,7 +100,7 @@ iex> DateTime.from_naive(~N[2016-05-24 13:26:08.003], "Etc/UTC")
 
 ## Working with timezones
 
-As we have noted in the previous chapter, by default Elixir does not have any timezone data.
+As we have noted in the previous section, by default Elixir does not have any timezone data.
```

Comment:
Word "chapter" may be a bit confusing — it may suggest previous lesson. In other places we use "section" and I think it's better.